app/controllers/api/v2/override_values_controller.rb

```diff
@@ -2,94 +2,37 @@ module Api
   module V2
     class OverrideValuesController < V2::BaseController
```

Comment:
@tbrisker This controller should get deleted though I believe, right?

Reply:
I think so, unless it's used by any of the plugins